machine-runner/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actyx/machine-runner",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
let’s call it 0.2.0 — there have been breaking API changes

Reply:
Changed to 0.2.0 in https://github.com/Actyx/machines/commit/0cb21a4e64fe127a9c20b9a3389ded703fae4500